lib/hammer_cli_foreman/compute_resource.rb

```diff
@@ -71,12 +71,21 @@ def print_data(data)
 
 
     class CreateCommand < HammerCLIForeman::CreateCommand
+      option "--public-key-path", "PUBLIC_KEY_PATH", _("Path to a file that contains oVirt public key (For oVirt only)"), :attribute_name => :option_public_key_path,
+             :required => false, :format => HammerCLI::Options::Normalizers::File.new
```

Comment:
I wonder why the `:required => false` and `:attribute_name => :option_public_key_path` are kept in the code. These are the default values and should have no impact on the code. Just making it more difficult to read. Is it intentional?